cpp/src/core/Transpose_Test/TransposeTest.cc

```diff
@@ -0,0 +1,34 @@
+// This file contains a simple tranpose function and a test to make sure that
```

Comment:
The test should directly test the transpose functions in cpu_operations.cc. We can discuss more in the lab.